### PR TITLE
(Proto) Remove returns from the requests

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

**Review by convoliution — CHANGES_REQUESTED** [review]
I think we should return an empty response (like we do [in `VersioningService`](https://github.com/VertaAI/modeldb/blob/master/protos/protos/public/modeldb/versioning/VersioningService.proto#L216)) instead of introducing `google/protobuf/empty.proto`.

It looks to me like `google/protobuf/empty.proto` causes problems during Swagger generation.
```
Processing ../../protos/gen/swagger/protos/public/modeldb/ExperimentRunService.swagger.json
Traceback (most recent call last):
  File "../tools/swagger_codegen.py", line 297, in <module>
    main()
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "../tools/swagger_codegen.py", line 43, in main
    create_api(result_dir, result_package, api_name, content, templates, file_suffix, case)
  File "../tools/swagger_codegen.py", line 95, in create_api
    success_response = resolve_type(definition)
  File "../tools/swagger_codegen.py", line 239, in resolve_type
    return resolve_type(typedef['schema'])
  File "../tools/swagger_codegen.py", line 241, in resolve_type
    if typedef['type'] == 'string':
KeyError: 'type'
```

**Comment by convoliution** [comment]
Adding John as a reviewer so frontend has visibility on this change

**Review by convoliution — APPROVED** [review]
Looks good to me. @kirillGulyaev has also confirmed that the WebApp will be fine, as well.

File: protos/protos/public/modeldb/ExperimentRunService.proto [hunk 1]
```diff
@@ -179,7 +176,6 @@ message AddExperimentRunTags {
     repeated string tags = 2;
 
     message Response {
-        ExperimentRun experiment_run = 1;
```
Comment (ravishetye) [inline protos/protos/public/modeldb/ExperimentRunService.proto]: revert

File: protos/protos/public/modeldb/ExperimentRunService.proto [hunk 2]
```diff
@@ -161,15 +160,13 @@ message UpdateExperimentRunName {
...
     }
 }
 message UpdateExperimentRunDescription {
     string id = 1;
     string description = 2;
 
     message Response {
-        ExperimentRun experiment_run = 1;
```
Comment (ravishetye) [inline protos/protos/public/modeldb/ExperimentRunService.proto]: revert

File: protos/protos/public/modeldb/ExperimentRunService.proto [hunk 3]
```diff
@@ -189,7 +185,6 @@ message DeleteExperimentRunTags {
     bool delete_all = 3;
 
     message Response {
-        ExperimentRun experiment_run = 1;
```
Comment (ravishetye) [inline protos/protos/public/modeldb/ExperimentRunService.proto]: revert

File: protos/protos/public/modeldb/ExperimentRunService.proto [hunk 4]
```diff
@@ -199,7 +194,6 @@ message AddExperimentRunTag {
     string tag = 2;
 
     message Response {
-        ExperimentRun experiment_run = 1;
```
Comment (ravishetye) [inline protos/protos/public/modeldb/ExperimentRunService.proto]: revert

File: protos/protos/public/modeldb/ExperimentRunService.proto [hunk 5]
```diff
@@ -208,7 +202,6 @@ message DeleteExperimentRunTag {
...
```
Comment (ravishetye) [inline protos/protos/public/modeldb/ExperimentRunService.proto]: revert

**Review by ravishetye — CHANGES_REQUESTED** [review]

File: protos/protos/public/modeldb/ExperimentRunService.proto (hunk 1 above)
Comment (anandjakhaniya) [reply protos/protos/public/modeldb/ExperimentRunService.proto]: Done

File: protos/protos/public/modeldb/ExperimentRunService.proto (hunk 2 above)
Comment (anandjakhaniya) [reply protos/protos/public/modeldb/ExperimentRunService.proto]: Done

File: protos/protos/public/modeldb/ExperimentRunService.proto (hunk 3 above)
Comment (anandjakhaniya) [reply protos/protos/public/modeldb/ExperimentRunService.proto]: Done

File: protos/protos/public/modeldb/ExperimentRunService.proto (hunk 4 above)
Comment (anandjakhaniya) [reply protos/protos/public/modeldb/ExperimentRunService.proto]: Done

File: protos/protos/public/modeldb/ExperimentRunService.proto (hunk 5 above)
Comment (anandjakhaniya) [reply protos/protos/public/modeldb/ExperimentRunService.proto]: Done

**Review by ravishetye — APPROVED** [review]

**Pull request merged**